girder/api/rest.py

```diff
@@ -34,6 +34,8 @@
 from girder.utility import config, JsonEncoder
 from six.moves import range, urllib
 
+READ_BUFFER_LEN = 65536
```

Comment:
You know me, I just like comments that say a little bit about why. Even if the answer is: "This is arbitrary."